configs/config.usa_baseline_2050.yaml

```diff
@@ -93,24 +94,57 @@ build_shape_options:
   gadm_url_prefix: "https://geodata.ucdavis.edu/gadm/gadm4.1/gpkg/"
 
 build_osm_network:  # Options of the build_osm_network script; osm = OpenStreetMap
-  group_tolerance_buses: 5000  # [m] (default 5000) Tolerance in meters of the close buses to merge
+  group_tolerance_buses: 10000  # [m] (default 5000) Tolerance in meters of the close buses to merge
```

Comment:
@SermishaNarayana could you please add a note here on why this change has been needed? Would be very helpful to keep track of things